### PR TITLE
Added docs for functions in `mobject_update_utils`

### PR DESCRIPTION
**Pull request by JasonGrace2282**
## Overview: What does this pull request change?

* Added docstrings for:
  * :meth:`always_rotate`
  * :meth:`always_shift`
  * :meth:`turn_animation_into_updater`
* Added typehints to above functions 

## Links to added or changed documentation pages

https://manimce--3325.org.readthedocs.build/en/3325/reference/manim.animation.updaters.mobject_update_utils.html

## Reviewer Checklist
- [ ] The PR title is descriptive enough for the changelog, and the PR is labeled correctly
- [ ] If applicable: newly added non-private functions and classes have a docstring including a short summary and a PARAMETERS section
- [ ] If applicable: newly added functions and classes are tested

### CODE REVIEW

File: manim/animation/updaters/mobject_update_utils.py [hunk 1]
```diff
@@ -99,7 +99,39 @@ def construct(self):
...
+
+                # always shift square one unit to the right
+                # even if there is an animation going on
+                always_shift(sq, RIGHT, rate=5)
+
+                self.add(sq)
+                self.play(tri.animate.set_fill(opacity=1))
+    """
+
     def normalize(v):
```
Comment (behackl) [inline manim/animation/updaters/mobject_update_utils.py:135]: This should be replaced by a call to `normalize` from `manim.utils.space_ops`.

File: manim/animation/updaters/mobject_update_utils.py [hunk 2]
```diff
@@ -99,7 +99,39 @@ def construct(self):
...
+    mobject: Mobject, direction: np.ndarray[np.float64] = RIGHT, rate: float = 0.1
+) -> Mobject:
+    """Shift the mobject by `rate*direction` every second
+
+    Parameters
+    ----------
+    mobject
+        The mobject to shift
+    direction
+        The direction to shift
```
Comment (behackl) [inline manim/animation/updaters/mobject_update_utils.py:112]:
```suggestion
        The direction to shift. The vector is normalized, the specified magnitude
        is not relevant.
```

File: manim/animation/updaters/mobject_update_utils.py [hunk 3]
```diff
@@ -99,7 +99,39 @@ def construct(self):
...
-def always_shift(mobject, direction=RIGHT, rate=0.1):
+def always_shift(
+    mobject: Mobject, direction: np.ndarray[np.float64] = RIGHT, rate: float = 0.1
+) -> Mobject:
+    """Shift the mobject by `rate*direction` every second
+
+    Parameters
+    ----------
+    mobject
+        The mobject to shift
```
Comment (behackl) [inline manim/animation/updaters/mobject_update_utils.py:110]:
```suggestion
        The mobject to shift.
```

File: manim/animation/updaters/mobject_update_utils.py [hunk 4]
```diff
@@ -99,7 +99,39 @@ def construct(self):
...
+    direction
+        The direction to shift
+    rate
+        Amount of direction to be covered in one second
```
Comment (behackl) [inline manim/animation/updaters/mobject_update_utils.py:114]:
```suggestion
        Length in Manim units which the mobject travels in one
        second along the specified direction.
```

File: manim/animation/updaters/mobject_update_utils.py [hunk 5]
```diff
@@ -99,7 +99,39 @@ def construct(self):
     return mob
 
 
-def always_shift(mobject, direction=RIGHT, rate=0.1):
+def always_shift(
+    mobject: Mobject, direction: np.ndarray[np.float64] = RIGHT, rate: float = 0.1
+) -> Mobject:
+    """Shift the mobject by `rate*direction` every second
```
Comment (behackl) [inline manim/animation/updaters/mobject_update_utils.py:105]:
```suggestion
    """A mobject which is continuously shifted along some direction
    at a certain rate.
```

File: manim/animation/updaters/mobject_update_utils.py [hunk 6]
```diff
@@ -99,7 +99,39 @@ def construct(self):
...
+    .. manim:: ShiftingSquare
+
+        class ShiftingSquare(Scene):
+            def construct(self):
+                sq = Square().set_fill(opacity=1)
+                tri = Triangle()
+                VGroup(sq, tri).arrange(LEFT)
+
+                # always shift square one unit to the right
+                # even if there is an animation going on
```
Comment (behackl) [inline manim/animation/updaters/mobject_update_utils.py:128]:
```suggestion
                # construct a square which is continuously
                # shifted to the right
```

File: manim/animation/updaters/mobject_update_utils.py [hunk 7]
```diff
@@ -110,18 +142,57 @@ def normalize(v):
...
 
 
-def always_rotate(mobject, rate=20 * DEGREES, **kwargs):
+def always_rotate(mobject: Mobject, rate: float = 20 * DEGREES, **kwargs) -> Mobject:
+    """Rotate the mobject by `rate` every second
+
+    Parameters
+    ----------
+    mobject
+        The mobject to rotate
```
Comment (behackl) [inline manim/animation/updaters/mobject_update_utils.py:151]:
```suggestion
    mobject
        The mobject to be rotated.
    rate
        The angle which the mobject is rotated by
        over one second.
    kwags
        Further arguments to be passed to :meth:`.Mobject.rotate`.
```

File: manim/animation/updaters/mobject_update_utils.py [hunk 8]
```diff
@@ -110,18 +142,57 @@ def normalize(v):
     return mobject
 
 
-def always_rotate(mobject, rate=20 * DEGREES, **kwargs):
+def always_rotate(mobject: Mobject, rate: float = 20 * DEGREES, **kwargs) -> Mobject:
+    """Rotate the mobject by `rate` every second
```
Comment (behackl) [inline manim/animation/updaters/mobject_update_utils.py:146]:
```suggestion
    """A mobject which is continuously rotated at a certain rate.
```

**Review by behackl — CHANGES_REQUESTED** [review]
Thanks for your efforts with documenting these! I've left some suggestions, please take a look -- otherwise, this looks fine to me.

File: manim/animation/updaters/mobject_update_utils.py (hunk 1 above)
Comment (JasonGrace2282) [reply manim/animation/updaters/mobject_update_utils.py]: done!

**Comment by JasonGrace2282** [comment]
@behackl done!

**Review by behackl — APPROVED** [review]

**Pull request merged**